include/NAS2D/Resources/Image.h

```diff
@@ -51,13 +52,15 @@ class Image : public Resource
 	~Image();
 
 	Vector<int> size() const;
+	Vector<int> center() const;
 
 	int width() const;
 	int height() const;
 
 	int center_x() const;
 	int center_y() const;
 
+	Color pixelColor(Point<int> point) const;
 	Color pixelColor(int x, int y) const;
```

Comment:
The split apart function overload is redundant.

Reply:
The split apart function still contains the actual implementation. The new overload just defers to the existing method. Would need more refactoring before the old method can be removed. Would also need to update all calls to it to use the new overload instead.

I recommend deferring until more refactoring can be done.